.github/workflows/build.yml

```diff
@@ -16,13 +16,17 @@ jobs:
             fail-fast: false
```

Comment:
Can you add Sylius version to run name like here https://github.com/BitBagCommerce/SyliusWishlistPlugin/blob/master/.github/workflows/build.yml?

Reply:
Done